### DISCUSSION TITLE
GIthub student developer pack activation

### DISCUSSION THREAD

**Discussion by bugnyx** [community/community#78741] [GitHub Education]
### Hi everyone,

Hello everyone,

I have recently applied for the student developer pack. I still have not received my benefits even though it says it was approved. From the other discussions it is my understanding that it usually takes 72 hours to get the benefits but it has already been 6 days for me as it got approved on December 6th. I just wanna know if something is wrong with my application and how long I have to wait before the benefits. 

Thanks for taking your time and reading this.
<img width="341" alt="Screenshot 2023-12-12 at 9 40 09 AM" src="https://github.com/community/community/assets/113361875/d180fa10-e577-44b5-9704-0a4985a01d1d">

**Comment by realSynced** [comment]
This same thing is currently happening to me. I sent in a support ticket yesterday so I could get more insight on this.

**Comment by Peter-K-Joseph** [comment]
Hey. Did you get the benefits yet?

**Comment by realSynced** [comment]
Yes.

**Comment by MohamedBoukthir** [comment]
it happened to me , no problem just wait or you should email the github
